wiremock/mappings/two_state_about_to_submit.json

```diff
@@ -0,0 +1,21 @@
+{
+  "request": {
+    "method": "POST",
+    "urlPath": "/case_type/two/state/about_to_submit"
```

Comment:
you should be able to reuse one of the previous files you already introduced, since the json response is identical

Reply:
This is a  quite different case which it would useful adding it. there is not a previous existing mock that emulate this response with 2 states.    I think that it would be nice having this test to cover all scenarios in Case Multiple Pages the integration test.